var/spack/repos/builtin/packages/vim/package.py

```diff
@@ -90,6 +97,17 @@ def install(self, spec, prefix):
         else:
             configure_args.append("--enable-rubyinterp=dynamic")
 
+        if '+lua' in spec:
+            configure_args.append("--enable-luainterp=yes")
+            configure_args.append("--with-lua-prefix=%s" % spec['lua'].prefix)
+        else:
+            configure_args.append("--enable-luainterp=dynamic")
```

Comment:
Hmm, I thought `--enable-XYZinterp=dynamic` (dynamic loading seems to be windows-only…) would be the default; but according to [this](https://github.com/vim/vim/blob/63de19e805a7df2b52ec0e705b6a668ecd8e1b64/src/configure.ac) `=no` seems to be the default.
I'll update the ~~changeset~~PR.